libnavigation-ui/src/test/java/com/mapbox/navigation/ui/routealert/TollCollectionAlertDisplayerTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.mapbox.navigation.ui.routealert
+
+import android.graphics.drawable.Drawable
+import com.mapbox.geojson.Point
+import com.mapbox.mapboxsdk.maps.Style
+import com.mapbox.mapboxsdk.style.layers.SymbolLayer
+import com.mapbox.mapboxsdk.style.sources.GeoJsonSource
+import com.mapbox.navigation.base.trip.model.alert.RestStopAlert
+import com.mapbox.navigation.base.trip.model.alert.RestStopType
+import com.mapbox.navigation.base.trip.model.alert.TollCollectionAlert
+import com.mapbox.navigation.base.trip.model.alert.TollCollectionType
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.unmockkObject
+import io.mockk.verify
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+
+class TollCollectionAlertDisplayerTest {
+
+    @Before
+    fun setUp() {
+        mockkObject(RouteAlertDisplayerProvider)
+        val tollCollectionsSource = mockk<GeoJsonSource>(relaxed = true)
+        val tollCollectionsLayer = mockk<SymbolLayer>(relaxed = true)
+        every {
+            RouteAlertDisplayerProvider.getGeoJsonSource(any())
+        } returns tollCollectionsSource
+        every {
+            RouteAlertDisplayerProvider.getSymbolLayer(any(), any())
+        } returns tollCollectionsLayer
+    }
+
+    @After
+    fun tearDown() {
+        unmockkObject(RouteAlertDisplayerProvider)
+    }
+
+    @Test
+    fun onNewRouteAlerts_withTollCollectionAlerts() {
+        val tollCollectionAlertDisplayer = createTollCollectionAlertDisplayer()
+
+        tollCollectionAlertDisplayer.onNewRouteAlerts(mockRouteAlertsWithTollCollectionAlerts())
+
+        val tollCollectionAlertUiModels = mutableListOf<List<TollCollectionAlertUiModel>>()
+        verify { tollCollectionAlertDisplayer.onNewRouteTollAlerts(capture(tollCollectionAlertUiModels)) }
```

Comment:
```
java.lang.AbstractMethodError: Method io/mockk/renamed/java/util/List$Subclass1.iterator()Ljava/util/Iterator; is abstract

Reply:
Something similar to https://github.com/mapbox/mapbox-navigation-android/pull/3685#issuecomment-732503877 happens here too. Note that neither `tollCollectionAlertDisplayer` nor `tollCollectionAlertUiModels` are mocks.